### ISSUE TITLE
Node position changes on `setItemState(... ,false)`

### ISSUE THREAD

**Issue by sklinov** [antvis/G6#2367]
- [ V ] I have searched the [issues](https://github.com/antvis/g6/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/antv-state-active-deactive-hhmp9?file=/index.js)

### Steps to reproduce
Hover and leave nodes to see the position of node border change (moves halfway right and down) while label stays it it's initial position.
It happens when `graph.setItemState(... , ... , false)` (or `graph.clearItemStates()`) is applied.

| Environment | Info |
|---|---|
| g6 | 4.0.2 |
| System | Windows 10 |
| Browser | Версия 87.0.4280.66 |

**Comment by sklinov** [comment]
And it looks like the problem goes away if we set item state to some other state:
```
graph.setItemState(item, fallbackState, true);
graph.setItemState(item, stateToBeSwtichedOff, false);
```
So is this a bug or it works the way it is expected (once set item must have some state) ?

**Comment by Heheroes** [comment]
Hello, I don't quite understand the problem you described. You didn't see any problems with your online presentation.
Could you give me a more detailed description？

**Comment by gk7261234** [comment]
- [ ] 
![企业微信截图_16067881858073](https://user-images.githubusercontent.com/19725081/100688143-93b65380-33bc-11eb-845d-3e1da380a965.png)
![企业微信截图_16067882004716](https://user-images.githubusercontent.com/19725081/100688145-95801700-33bc-11eb-95bb-dd77ff3d344f.png)

**Comment by Heheroes** [comment]
Please post your bug code to the demo platform.

**Comment by sklinov** [comment]
@Heheroes 
> Please post your bug code to the demo platform.

Here's the link (it's the same one as in the original post).
https://codesandbox.io/s/antv-state-active-deactive-hhmp9?file=/index.js

And here's the demo:
![Antv g6 node state bug](https://user-images.githubusercontent.com/38185977/100692970-c0af3a00-33ad-11eb-868a-40ca7e78adce.gif)

**Comment by Yanyan-Wang** [comment]
It will be fixed in 4.0.3 soon.

**Comment by Yanyan-Wang** [comment]
Fixed by 4.0.3.
